### DISCUSSION TITLE
Bernardo Fernandes

### DISCUSSION THREAD

**Discussion by Bennylave** [aiqc/AIQC#49] [Introduction]
Hi there! My name is Bernardo Fernandes and I'm from Lisbon, Portugal.

I am a Data Engineer at the moment and I mostly work with Hadoop (HDFS, Hive, Impala, Spark etc.). In my free time I like to do side projects that involve Machine Learning and working with Pandas. My mostly used languages are Python, Scala and SQL.

I enjoy travelling, sports (basketball, football, tennis), cooking and taking a nice walk in the nature.

**Comment by aiqc** [comment]
What's your favorite club team? The old benfica sponsor used to crack me up

![image](https://user-images.githubusercontent.com/74990642/113477855-fd219e00-9452-11eb-977b-05baac292e28.png)

**Reply by Bennylave** [reply]
It's Sporting Clube de Portugal. Ronaldo's first professional team. But Benfica is a good guess. You would be 50% of the time right!

**Comment by amirhszd** [comment]
Hey Bernardo, welcome to the gang!

**Reply by Bennylave** [reply]
Thanks! Can't wait to contribute to AIQC. Cheers
